Protocol/Tests/Protocol/Params/Param/Interprete/Exceptions/Exception/Display/CheckStateAttribute.cs

```diff
@@ -0,0 +1,162 @@
+namespace Skyline.DataMiner.CICD.Validators.Protocol.Tests.Protocol.Params.Param.Interprete.Exceptions.Exception.Display.CheckStateAttribute
+{
+    using System;
+    using System.Collections.Generic;
+    using Skyline.DataMiner.CICD.Models.Protocol.Edit;
+    using Skyline.DataMiner.CICD.Models.Protocol.Enums;
+    using Skyline.DataMiner.CICD.Models.Protocol.Read;
+    using Skyline.DataMiner.CICD.Validators.Common.Interfaces;
+    using Skyline.DataMiner.CICD.Validators.Common.Model;
+    using Skyline.DataMiner.CICD.Validators.Protocol.Common;
+    using Skyline.DataMiner.CICD.Validators.Protocol.Common.Attributes;
+    using Skyline.DataMiner.CICD.Validators.Protocol.Common.Extensions;
+    using Skyline.DataMiner.CICD.Validators.Protocol.Generic;
+    using Skyline.DataMiner.CICD.Validators.Protocol.Interfaces;
+
+    [Test(CheckId.CheckStateAttribute, Category.Param)]
+    internal class CheckStateAttribute : IValidate, ICodeFix, ICompare
```

Comment:
ICompare not being used, remove the interface.

Reply:
Comment it out (same with the method) as mentioned in the c# comment below